src/main/java/core/basesyntax/service/StorageServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class StorageServiceImpl implements StorageService {
+    private List<FruitTransaction> fruitTransactionList;

```

Comment:
it's better to pass transaction as method param, so some service can process different lists of transactions